tests/Integration/Integrations/Predis/PredisTest.php

```diff
@@ -48,6 +49,19 @@ public function testPredisConstruct()
             $this->assertNotNull($client);
         });
 
```

Comment:
In general we prefer (and try to enforce) tests that only test one specific thing. This way it is clear what we may "miss" of we, at a certain point", remove that test. 😄 

In this case I would create a new test instead of editing the current one. Specifically:

 - I would rename `testPredisConstruct` to `testPredisConstructOptionsAsArray`
 - Create an exact same replica `testPredisConstructOptionsAsObject` which contains the part that you added to the existing test.

This way we know exactly what we miss if we remove one of the two.

Reply:
Of Course, I'll do it soon

Reply:
Done!